srcpkgs/notmuch/patches/cross.patch

```diff
@@ -1,6 +1,20 @@
---- configure.orig
-+++ configure
-@@ -370,7 +370,7 @@
+From 7bb539777d688acd5ca6081919047b60d47566d3 Mon Sep 17 00:00:00 2001
+From: Doan Tran Cong Danh <congdanhqx@gmail.com>
+Date: Mon, 10 Jun 2019 20:30:11 +0700
```

Comment:
I suppose that you've created a patch against configure.in and then configure was regenerated, isn't it? otherwise the patch needs to be made each time a new version is released.

Reply:
No, `notmuch` has neither `configure.in` or anythings for autotools (`autoconf`).

They only provide `configure` script.
Here is the original `configure` script in their github mirror https://github.com/notmuch/notmuch/blob/master/configure

Reply:
In that case seems ok to me, thanks.

Reply:
I need to make the patch against `configure` in this release
because they've added a lot of compile-and-test for gpgme, xapian and zlib.